src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
+import {Add as AddIcon} from "@material-ui/icons";
+
+
+
+interface AddConceptToDictionaryIconProps {
+    dictionaryToAddTo:string;
+    canAddConcept:string;
+    row:APIConcept|undefined ;
+    addConceptsToDictionary: Function;
+   
+}
+
+
+    export function AddConceptToDictionaryIcon(props: AddConceptToDictionaryIconProps) {
+        const {
+          dictionaryToAddTo,
+          canAddConcept,
+          row,
+          addConceptsToDictionary
+        } = props;
+    return (canAddConcept=="false"  || dictionaryToAddTo==null || dictionaryToAddTo==undefined)? (null):
```

Comment:
Please use `===` instead of `==`. Also, at least per the typing, `dictionaryToAddTo` cannot possibly be `null` or `undefined` at this point. Last point on this: if you're going to check if something is `null` or `undefined`, check the `undefined` first.

```ts
return (!canAddConcept || dictionaryToAddTo === undefined || dicationaryToAddTo === null) ? null : ( ...
```

Reply:
thank you @ibacher this was a blunder
`dictionaryToAddTo cannot possibly be null or undefined at this point.`
it can be this is why the previous submission was [failing to build The Travis CI](https://talk.openmrs.org/t/i-dont-know-how-to-tackle-the-travis-ci-build-failed/31763).

Reply:
Then we need to change the typing of `dictionaryToAddTo` to something like `dictionaryToAddTo?: string` to reflect that.